run_benchmark.py

```diff
@@ -102,20 +113,26 @@ def benchmark(data, settings):
         "lr": 0.005,
         "epochs": 50,
         "act": "elu",
+        "out_act": "relu",
         "batch_size": 32,
         "loss": "mae",
         "xscale": "minmax",
     }
+
+    #best_settings = None
     names = [[[field for field in data.df_targets.columns]]]
     weights = {field: 1 for field in data.df_targets.columns}
     return matbench_benchmark(
         data,
         names,
         weights,
         best_settings,
+        model_type="Ensemble_MODNetModel",
```

Comment:
```suggestion
        model_type="Ensemble_MODNetModel",
```

Tweak this to the new model_type we've added